charts/agent/Chart.yaml

```diff
@@ -30,4 +30,4 @@ sources:
 - https://app.sysdigcloud.com/#/settings/user
 - https://github.com/draios/sysdig
 type: application
-version: 1.23.3
+version: 1.23.4
```

Comment:
Bump to minor

Reply:
I've seen it being bumped to patch so far for similar changes, is there a guideline I can refer to?